src/sql/src/query.rs

```diff
@@ -3076,7 +3109,6 @@ pub fn plan_cast_implicit<'a>(
     let from_scalar_type = ecx.column_type(&expr).scalar_type;
 
     match (&from_scalar_type, &to_scalar_type) {
-        (String, String) | (Unknown, String) => {}
```

Comment:
You'll need to re-add this `(String, String)` line for the time being; I have a refactor for the implicit casts in mind but will wait for this to land.

Reply:
Oh crap. Did not mean to delete the `(String, String)` bit.